plugins/MauticSocialBundle/Helper/TwitterCommandHelper.php

```diff
@@ -336,7 +337,9 @@ public function createLeadsFromStatuses($statusList, $monitor)
                 $leadEntity->setPreferredProfileImage('Twitter');
 
                 // save the lead now
-                $leadEntity->setLastActive($lastActive->format('Y-m-d H:i:s'));
+                if ($lastActive instanceof DateTime) {
```

Comment:
@kuzmany just looking at this code... Did you really wanted it should be instance of `Symfony\Component\Validator\Constraints\DateTime` because that class doesn't have the `format` method? Shouldn't it be `\DateTimeInterface` instead?

Reply:
@kuzmany have you seen the note above?

Reply:
I think have to be \DateTime  set few lines before
https://github.com/kuzmany/mautic/blob/f8509b83cbb13bb5d81bf1111d61147f3647c48d/plugins/MauticSocialBundle/Helper/TwitterCommandHelper.php#L291

What do you think? Should I send new PR or can we update it on staging directly?

Reply:
Yes, I think it is instance of `\DateTime`.

But we use [\DateTime](http://php.net/manual/en/class.datetimeimmutable.php) with the new code wherever possible because it's safer. Both `\DateTime` and `DateTimeImmutable` implements [\DateTimeInterface](http://php.net/manual/en/class.datetimeinterface.php) so that's what I'd like to use for the `instanceof` comparison.

Create new PR please.